README.md

```diff
@@ -239,6 +240,94 @@ This already goes with how CSP is currently enforcing its rules canonically in t
+Thus, the information it naturally exposes is regarding when a new same origin realm is introudced under the execution environment of the top realm of the website.
+
+> 02.  Do features in your specification expose the minimum amount of information necessary to implement the intended functionality?
+
+Yes, I believe so - no further information is being provided to the website other than what's described under Q#1
+
+> 03.  Do the features in your specification expose personal information, personally-identifiable information (PII), or information derived from either?
+
+No
+
+> 04.  How do the features in your specification deal with sensitive information?
+
+They have nothing to do with such information
+
+> 05.  Does data exposed by your specification carry related but distinct information that may not be obvious to users?
+
+The only distinct information that's being provided to the user that was not possible before is when same origin realms are introduced into the website's execution environment
+
+> 06.  Do the features in your specification introduce state that persists across browsing sessions?
+
+No
+
+> 07.  Do the features in your specification expose information about the underlying platform to origins?
+
+No
+
+> 08.  Does this specification allow an origin to send data to the underlying platform?
+
+The answer is basically no aside for the value of the configured new CSP directive which should be parsed into a URL of a remote resource
+
+> 09.  Do features in this specification enable access to device sensors?
+
+No
+
+> 10.  Do features in this specification enable new script execution/loading mechanisms?
+
+Yes, this feature focuses on allowing a website to register JavaScript code to be loaded within new realms when are introduced into the execution environment of the website at runtime.
+While the browser somewhat knows already how to load JavaScript code within new realms with features such as web extensions' `content_script:run_at`, granting such power to websites (rather than extensions) is necessarily new.
+
+> 11.  Do features in this specification allow an origin to access other devices?
+
+No
+
+> 12.  Do features in this specification allow an origin some measure of control over a user agent's native UI?
+
+No
+
+> 13.  What temporary identifiers do the features in this specification create or expose to the web?
+
+None
+
+> 14.  How does this specification distinguish between behavior in first-party and third-party contexts?
+
+This feature expects the website to provide a URL to a remote JavaScript file to be loaded within new same origin realms only, so naturally there's an important distinction where the file should only be loaded and executed within first-party contexts and not third-party contexts. Telling them apart should (hopefully) be rather straight forward since CSP is already very good at this exactly.
```

Comment:
IIUC, the script would only run in same-origin contexts (which is stricter than first-party/same-site contexts)

Reply:
Of course, but to answer the question, isn't that a distinction we are expected to manifest in the specification? Isn't this question asking us to clarify how the 1st vs 3rd party distinction is relevant here?

Reply:
"This feature enables sites to run a remote JavaScript file in the context of new same-origin realms, and only such realms. As same-origin is stricter than same-site, the distinction between first-party and third-party contexts falls out of that restriction."?

Reply:
Only now realizing how "party" refers to "site"s as opposed to "origin"s - on it

Reply:
67cdc362badc935f0c7928e8312804de476d9fd8